### ISSUE TITLE
Menu button doesn't need a pirate aria label (even though it's cool) and should be nested in a `nav`

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org-11ty#135]
## Description

Two issues exist, one is easy and the other is a _little_ complex

### The easy one (deleting an aria-label)

The menu still has an `aria-label` on pirates (which I love, but is a bit misleading). Since this is labelled, it doesn't need an `aria-label` at all fortunately!

<img width="500" alt="Screen Shot 2022-09-25 at 8 32 19 PM" src="https://user-images.githubusercontent.com/14154792/192173149-c1880b15-6b8a-4073-9e9b-6ccafc96c240.png">

### The :avril-lavigne: (maybe) complicated one? (inserting a `nav` element)

Currently, the website doesn't have a `nav` landmark, which will appear in a screen reader user's landmarks list as a place to go to for navigation.

Ideally, the code should be structured so the `nav` sits within the `header` like this

```
<header>
  <nav>
    <!-- ye old list of navigation items here -->
  </nav>
</header>
```

It could be a bit awkward that the menu button will be nested within a navigation landmark with the home link _outside_ that menu button's list of links, but it's definitely not a major (or even moderate) issue.

For now I'd recommend we try to just add in that `nav` for our nice semantic landmarks.
